idemp-base/src/org/bandahealth/idempiere/base/model/MBandaSetup.java

```diff
@@ -1359,6 +1360,40 @@ public boolean configureClientUsers() {
 		});
 		return true;
 	}
+	
+	/**
+	 * Create default business partners for new clients
+	 * @return
+	 */
+	public boolean createDefaultBusinessPartners() {
+		List<MBPartner_BH> patients = new Query(this.context, MBPartner_BH.Table_Name,
+				MBPartner_BH.COLUMNNAME_AD_Client_ID + "=?", getTransactionName()).setParameters(MClient_BH.CLIENTID_CONFIG)
+				.list();
+		patients.forEach((patient) -> {
+			if (!patient.save()) {
+				log.warning("Failure: Could not save default patient");
+			}
+		});
+		
+		return true;
+	}
+	
+	/**
+	 * Create default business partner groups for new clients
+	 * @return
+	 */
+	public boolean createDefaultBusinessPartnerGroups() {
+		List<MBPGroup> bpGroups = new Query(this.context, MBPGroup.Table_Name,
+				MBPGroup.COLUMNNAME_AD_Client_ID + "=?", getTransactionName()).setParameters(MClient_BH.CLIENTID_CONFIG)
```

Comment:
I can see the change here. Strange

Reply:
Yes, I can see the change for the different client config, but not renaming the variable in the line above to avoid abbreviations.